### PR TITLE
OSSMDOC-374: Jaeger 1.24.0 Release Notes.

### PR DESCRIPTION
**Pull request by JStickler**
Release notes for the Jaeger 1.24 release (new features, known issues, fixed issues).

Added link to the Logging upgrade instructions for Elasticsearch.

Updated the Elasticsearch install instructions to point to 5.x channel.

Preview link for RN is here -> https://deploy-preview-35210--osdocs.netlify.app/openshift-enterprise/latest/jaeger/rhbjaeger-release-notes.html

### CODE REVIEW

**Comment by JStickler** [comment]
@iblancasa

**Review by jpkrohling — APPROVED** [review]

**Review by iblancasa — APPROVED** [review]

File: modules/jaeger-rn-new-features.adoc [hunk 1]
```diff
@@ -9,6 +9,20 @@ Result – If changed, describe the current user experience.
 ////
 
 [id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 1.24.0
+
+This release of {ProductName} includes the following new features:
+
+* Adds support for the Red Hat Elasticsearch Operator 5.0.
```
Comment (jkandasa) [inline modules/jaeger-rn-new-features.adoc:16]: @JStickler we have tested in ESO 5.0 and in 5.1

Comment (JStickler) [reply modules/jaeger-rn-new-features.adoc]: Updated!

**Review by jkandasa — APPROVED** [review]

File: jaeger/jaeger_install/rhbjaeger-updating.adoc [hunk 2]
```diff
@@ -10,3 +10,5 @@ The OLM queries for available Operators as well as upgrades for installed Operat
 For more information about how {product-title} handled upgrades, refer to the xref:../../operators/understanding/olm/olm-understanding-olm.adoc#olm-understanding-olm[Operator Lifecycle Manager] documentation.
 
 The update approach used by the Jaeger Operator upgrades the managed Jaeger instances to the version associated with the Operator. Whenever a new version of the Jaeger Operator is installed, all the Jaeger application instances managed by the Operator will be upgraded to the Operator's  version.  For example, if version 1.10 is installed (both Operator and backend components) and the Operator is upgraded to version 1.11, then as soon as the Operator upgrade has completed, the Operator will scan for running Jaeger instances and upgrade them to 1.11 as well.
```
Comment (rolfedh) [inline jaeger/jaeger_install/rhbjaeger-updating.adoc:12]:
```suggestion
The update approach used by the Jaeger Operator upgrades the managed Jaeger instances to the version associated with the Operator. Whenever a new version of the Jaeger Operator is installed, all the Jaeger application instances managed by the Operator will be upgraded to the Operator's version.  For example, if version 1.10 is installed (both Operator and backend components) and the Operator is upgraded to version 1.11, then as soon as the Operator upgrade has completed, the Operator will scan for running Jaeger instances and upgrade them to 1.11 as well.
```

File: modules/jaeger-rn-known-issues.adoc [hunk 3]
```diff
@@ -23,6 +23,13 @@ These limitations exist in Jaeger:
 
 These are the known issues in Jaeger:
 
+* link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057] The Kafka API has been updated to `v1beta2` to support Strimzi Kafka Operator 0.23.0.  However this API version is not supported by AMQ Streams 1.6.3.  If you have the following environment your Jaeger services will not be upgraded and you cannot create new Jaeger services and cannot modify existing Jaeger services:
```
Comment (rolfedh) [inline modules/jaeger-rn-known-issues.adoc:26]:
```suggestion
* link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057] The Kafka API has been updated to `v1beta2` to support Strimzi Kafka Operator 0.23.0.  However, this API version is not supported by AMQ Streams 1.6.3.  If you have the following environment, your Jaeger services will not be upgraded, and you cannot create new Jaeger services and cannot modify existing Jaeger services:
```

File: modules/jaeger-rn-new-features.adoc [hunk 4]
```diff
@@ -9,6 +9,20 @@ Result – If changed, describe the current user experience.
...
 [id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 1.24.0
+
+This release of {ProductName} includes the following new features:
+
+* Adds support for the Red Hat Elasticsearch Operator 5.0 and 5.1.
+
+* Adds Elasticsearch rollover support.
+
+* Adds Elasticsearch tolerations support.
```
Comment (rolfedh) [inline modules/jaeger-rn-new-features.adoc:20]: Do we need "Adds"? It seems redundant to "the following new features:"

File: modules/jaeger-rn-new-features.adoc [hunk 5]
```diff
@@ -9,6 +9,20 @@ Result – If changed, describe the current user experience.
...
+
+* Updates the Jaeger Operator to include support for Strimzi Kafka Operator 0.23.0.  There is a known issue and workaround associated with this update, documented in the Known Issues section.
```
Comment (rolfedh) [inline modules/jaeger-rn-new-features.adoc:22]:
```suggestion
* Updates the Jaeger Operator to include support for Strimzi Kafka Operator 0.23.0.  A known issue and workaround associated with this update is documented in the "Known Issues" section.
```

**Review by rolfedh — APPROVED** [review]
Some minor suggestions. Otherwise, LGTM.

File: modules/jaeger-rn-fixed-issues.adoc [hunk 6]
```diff
@@ -15,6 +15,10 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
+* link:https://issues.redhat.com/browse/TRACING-2009[TRACING-2009] The Jaeger Operator has been updated to include support for Strimzi Kafka Operator 0.23.0.
+
+* link:https://issues.redhat.com/browse/TRACING-1907[TRACING-1907] The Jaeger agent sidecar injection was failing due to missing config maps in the application namespace. The config maps were getting automatically deleted due to an incorrect `OwnerReference` field setting and as a result, the application pods were not moving past the "ContainerCreating" stage. The incorrect settings have been removed.
```
Comment (jeana-redhat) [inline modules/jaeger-rn-fixed-issues.adoc:20]: Wondering if ContainerCreating should be in backticks or bold? Not sure where (if anywhere) that text appears so don't know, just seems like a thing that might be in backticks or bold :)

File: modules/jaeger-rn-known-issues.adoc [hunk 7]
```diff
@@ -23,6 +23,13 @@ These limitations exist in Jaeger:
 
 These are the known issues in Jaeger:
 
-* link:https://bugzilla.redhat.com/show_bug.cgi?id=1918920[BZ-1918920] The Elasticsearch pods does not get restarted automatically after an update.  As a workaround, restart the pods manually.
+* link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057] The Kafka API has been updated to `v1beta2` to support Strimzi Kafka Operator 0.23.0.  However, this API version is not supported by AMQ Streams 1.6.3. If you have the following environment, your Jaeger services will not be upgraded, and you cannot create new Jaeger services and cannot modify existing Jaeger services:
```
Comment (jeana-redhat) [inline modules/jaeger-rn-known-issues.adoc:26]:
```suggestion
* link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057] The Kafka API has been updated to `v1beta2` to support Strimzi Kafka Operator 0.23.0.  However, this API version is not supported by AMQ Streams 1.6.3. If you have the following environment, your Jaeger services will not be upgraded, and you cannot create new Jaeger services or modify existing Jaeger services:
```

File: modules/jaeger-rn-known-issues.adoc [hunk 8]
```diff
@@ -23,6 +23,13 @@ These limitations exist in Jaeger:
...
 
-* link:https://issues.redhat.com/browse/TRACING-809[TRACING-809] Jaeger Ingester is incompatible with Kafka 2.3. When there are two or more instances of the Jaeger Ingester and enough traffic it will continuously generate rebalancing messages in the logs.  This is due to a regression in Kafka 2.3 that was fixed in Kafka 2.3.1.  For more information, see https://github.com/jaegertracing/jaeger/issues/1819[Jaegertracing-1819].
+** Jaeger Operator channel: *1.17.x stable* or *1.20.x stable*
+** AMQ Streams Operator channel: *amq-streams-1.6.x*
++
+The resolve this issue, switch the subscription channel for your AMQ Streams Operator to either *amq-streams-1.7.x* or *stable*.
```
Comment (jeana-redhat) [inline modules/jaeger-rn-known-issues.adoc:31]:
```suggestion
To resolve this issue, switch the subscription channel for your AMQ Streams Operator to either *amq-streams-1.7.x* or *stable*.
```

File: modules/jaeger-rn-new-features.adoc [hunk 9]
```diff
@@ -3,12 +3,26 @@ Module included in the following assemblies:
...
+
+This release of {ProductName} includes the following new features:
+
+* Adds support for the Red Hat Elasticsearch Operator 5.0 and 5.1.
+
+* Adds Elasticsearch rollover support.
+
+* Adds Elasticsearch tolerations support.
+
+* Updates the Jaeger Operator to include support for Strimzi Kafka Operator 0.23.0. A known issue and workaround associated with this update is documented in the "Known Issues" section.
```
Comment (jeana-redhat) [inline modules/jaeger-rn-new-features.adoc:22]: Is there a reason that mostly the pattern "the [whatever] Operator" is used, but for Strimzi Kafka Operator the article "the" is omitted? (Not just here, in all instances)

**Review by jeana-redhat — COMMENTED** [review]
A couple suggestions and also a couple questions :+1:

File: modules/jaeger-rn-fixed-issues.adoc (hunk 6 above)
Comment (JStickler) [reply modules/jaeger-rn-fixed-issues.adoc:20]: From the ticket I think it's a status message.  Do messages that appear in the console count as code? 🤔

File: modules/jaeger-rn-new-features.adoc (hunk 9 above)
Comment (JStickler) [reply modules/jaeger-rn-new-features.adoc:22]: Fixed.

File: modules/jaeger-rn-fixed-issues.adoc (hunk 6 above)
Comment (jeana-redhat) [reply modules/jaeger-rn-fixed-issues.adoc:20]: Typically, yes, messages in the GUI go in backticks. But if you're not 100% sure how/where this appears, I can't really see if being critical :)

**Pull request merged**